.github/workflows/ci.yml

```diff
@@ -65,7 +65,7 @@ jobs:
     steps:
       - run: 'echo prechecks complete'
 
-  # Build github-metrics-aggregator-webhook and push to artifact registry
+  # Build gmactl and push to artifact registry
```

Comment:
Nit: I dislike "ctl" commands. Can we just name this `github-metrics-aggregator` or `ghma`?

Reply:
ack was looking to be consistent but i agree, going with the former.